ItAcademy/src/myProjects/Task_1/Part_1_2/AppForExecutors.java

```diff
@@ -9,13 +9,13 @@
 
 public class AppForExecutors {
     public static void main(String[] args) {
```

Comment:
Please remove logic from main() and refactor your methods to make them shorter and easier to understand.